### PR TITLE
#11 - Fix less imports for flow

### LINKED ISSUE #11: Flow fails on LESS files
1) Use `import` to bring in a LESS file (`import './styles.less';`)
2) Run `yarn run lint:flow`

Observe error stating `Cannot resolve module './styles.less'.`

---

The `.flowconfig` file is set up to ignore `.*/.less$`, and there's also a `name_mapper` set up to parse LESS files as empty objects. Neither of these seem to remedy the issue.

### PR DESCRIPTION
**Pull request by andymikulski**
Fixes #11 

- Adds `NullModule` file needed for flow
- Corrects .flowconfig around the `less` imports

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -9,6 +9,7 @@
 [libs]
 
 [options]
+module.name_mapper.extension='less' -> '<PROJECT_ROOT>/NullModule.js'
 module.name_mapper='^console\/\(.*\)$' -> '<PROJECT_ROOT>/console/\1'
-module.name_mapper='.*\(.less\)$' -> 'empty/object'
```
Comment (magopian) [inline .flowconfig]: I believe this would have worked if we had a dependency on https://www.npmjs.com/package/empty

Comment (andymikulski) [reply .flowconfig]: Yeah, I tried that after some searching, but it didn't work. Flow has been a real pain around "things that should work but don't, and no warnings or errors are thrown."

**Comment by andymikulski** [comment]
I've changed the name to `FlowStubModule` (so it's easy to guess what it is for at first glance), and added a clarifying comment to the top of the file.

File: FlowStubModule.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/**
+ * This is a stub file used by the Flow checker for filetypes we wish to ignore.
+ * Essentially, this tells Flow that the module does not export anything, so
+ * there is nothing for it to type check.
+ */
```
Comment (magopian) [inline FlowStubModule.js]: 💯

**Pull request merged**